runtime/doc/lua.txt

```diff
@@ -801,6 +801,28 @@ vim.region({bufnr}, {pos1}, {pos2}, {type}, {inclusive})       *vim.region()*
         whether the selection is inclusive or not, into a zero-indexed table 
         of linewise selections of the form `{linenr = {startcol, endcol}}` .
 
+vim.register_keystroke_callback({id}, {fn})  *vim.register_keystroke_callback()*
+        Register a lua {fn} with an {id} to be run after every keystroke.
+
+        Callbacks will not be run unless 'keystrokecallback' is set.
+
+        NOTE: {fn} will be automatically removed if an error occurs while
+        calling.
+
+        Parameters: ~
+            {id}: (string)  Name of the ID.
```

Comment:
Yeah, that seems better. I'll change it to a namespace. That way it will make more sense to clear as well, since you're clearing from the namespace of your app.